ext/proxy/Dxxxx.md

```diff
@@ -289,8 +289,8 @@ template <Readable I1, Readable I2>
   requires !Swappable<reference_t<I1>, reference_t<I2>>() &&
     IndirectlyMovable<I1, I2>() && IndirectlyMovable<I2, I1>()
```

Comment:
These need to be `Storable` to support construction of `value_type_t<In>` from `rvalue_reference_t<In>`.